docker/jupyter/scripts/extension/install_TOC.sh

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```

Comment:
I think this script needs more :)

Reply:
this is what happens when you work during the plenary :)